### ISSUE TITLE
Abandoned repository?

### ISSUE THREAD

**Issue by zerodayyy** [MongoEngine/extras-mongoengine#27]
Hi, I wanted to use this project as I need Enum support for MongoEngine, but this repository doesn't seem to see much activity. Is it abandoned?

**Comment by gryphon2411** [comment]
I would consider it abandoned, unless specified otherwise.
The latest commit was 4 years ago.

**Comment by roysmith** [comment]
I had done some work there a long time ago, but have no future plans.  I'm fine with abandoning it.

**Comment by terencehonles** [comment]
@roysmith I don't think we want it abandoned, we want someone to look at the PRs and potentially merge them (I've proposed a few). I'm not sure who might have the capacity to maintain this repo, but it would be nice if we looked for a maintainer.

**Comment by terencehonles** [comment]
Or should we look at merging the relevant bits of this repo into https://github.com/MongoEngine/mongoengine so this repo isn't needed anymore?
